x/service/types/relay.go

```diff
@@ -0,0 +1,23 @@
+package types
```

Comment:
OPTIONAL - LMK WYT

I can see you documented the side effect of nullifying the signature when retrieving the signable bytes but....

Wdyt of storing the signature, marshaling and avoiding side-effect altogether?

e.g.

```go
sigTemp = req.Meta.Signature
req.Meta.Signature = nil
signableBz := req.Marshal()
req.Meta.Signature = sigTemp
return signableBz
```

Reply:
There is no side affect as this is not a pointer reciever and so this is ultimately added steps for something that doesnt change the signature field anyway

Reply:
![Uploading giphy (3).gif…]()

Reply:
![giphy (3)](https://github.com/pokt-network/poktroll/assets/1892194/414ef518-19fb-408f-86cc-8adf7a0409c1)